deployment/deployment-template.yaml

```diff
@@ -182,6 +182,9 @@ spec:
             # Enable mattermost endpoint
             - name: MATTERMOST_ENDPOINT
               value: ""
+            # Enable MS Teams webhook endpoint
+            - name: TEAMS_WEBHOOK_URL
+              value: "{{ .teams_webhook_url }}"
```

Comment:
Non-existent value elsewhere. Seemingly this file should be removed or its associated values.yaml should be restored.